geode-core/src/main/java/org/apache/geode/internal/cache/execute/InternalFunctionExecutionServiceImpl.java

```diff
@@ -116,6 +117,11 @@ public Execution onRegion(Region region) {
       throw new FunctionException("Region instance passed is null");
     }
 
+    if (region.getAttributes() == null) {
```

Comment:
How about check if (region.isDestroyed()) here?
And then throw FunctionException that region is destroyed?

Reply:
fixed